### PR TITLE
[VEN-2552]: add TUSD market to core pool on ethereum

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

Sepolia multisig tx: https://sepolia.etherscan.io/tx/0x3560bd0036c319690a822330048cac0fc936269261a881d3fd22e8b82994cbc1#eventlog

Resolves VEN-2552

### CODE REVIEW

File: multisig/proposals/sepolia/vip-019/index.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import { ethers } from "hardhat";
+
+import { NETWORK_ADDRESSES } from "../../../../src/networkAddresses";
+import { makeProposal } from "../../../../src/utils";
+
+const { sepolia } = NETWORK_ADDRESSES;
+
+export const TUSD = "0x78b292069da1661b7C12B6E766cB506C220b987a";
+export const vTUSD = "0xE23A1fC1545F1b072308c846a38447b23d322Ee2";
+export const INITIAL_SUPPLY = parseUnits("1000000", 18);
```
Comment (chechu) [inline multisig/proposals/sepolia/vip-019/index.ts:11]:
```suggestion
export const INITIAL_SUPPLY = parseUnits("5000", 18);
```

File: multisig/proposals/ethereum/vip-021/index.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import { makeProposal } from "../../../../src/utils";
+
+const { ethereum } = NETWORK_ADDRESSES;
+
+export const CHAINLINK_TUSD_FEED = "0xec746eCF986E2927Abd291a2A1716c940100f8Ba";
+const CHAINLINK_STALE_PERIOD_26H = 26 * 60 * 60; // 26 hours
+
+export const TUSD = "0x0000000000085d4780B73119b644AE5ecd22b376";
+export const vTUSD = "0x13eB80FDBe5C5f4a7039728E258A6f05fb3B912b";
+export const INITIAL_SUPPLY = parseUnits("1000000", 18);
```
Comment (chechu) [inline multisig/proposals/ethereum/vip-021/index.ts:14]:
```suggestion
export const INITIAL_SUPPLY = parseUnits("5000", 18);
```

File: multisig/proposals/sepolia/vip-019/index.ts (hunk 1 above)
Comment (GitGuru7) [reply multisig/proposals/sepolia/vip-019/index.ts]: https://github.com/VenusProtocol/vips/pull/267/commits/425ff7a4dfde46d0e741a2c4a42136ee1a687ca8

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/296?chainId=56

**Comment by chechu** [comment]
https://etherscan.io/tx/0x0b7d1f5bf7a8af37303ee20a8ba29a34926b1540628b8692b28142c0493c2922

**Pull request merged**